addons/account/models/partner.py

```diff
@@ -538,6 +538,11 @@ def _get_company_currency(self):
     duplicated_bank_account_partners_count = fields.Integer(
         compute='_compute_duplicated_bank_account_partners_count',
     )
+    is_accounting_fields_required = fields.Boolean(compute="_compute_is_accounting_fields_required")
```

Comment:
```suggestion
    is_coa_installed = fields.Boolean(compute="_compute_is_accounting_fields_required")
```
I think it makes more sense compared to what the field is actually used for :nerd_face: